opensrp-reveal/src/test/java/org/smartregister/reveal/view/ListTasksActivityTest.java

```diff
@@ -0,0 +1,32 @@
+package org.smartregister.reveal.view;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.robolectric.Robolectric;
+import org.smartregister.Context;
+import org.smartregister.reveal.BaseUnitTest;
+
+import java.util.ArrayList;
+
+import static org.junit.Assert.assertNotNull;
+
+/**
+ * Created by samuelgithengi on 1/23/20.
+ */
+public class ListTasksActivityTest extends BaseUnitTest {
+
+
+    private ListTasksActivity listTasksActivity;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'listTasksActivity' could be replaced by a local variable.](https://app.codacy.com/manual/OpenSRP/opensrp-client-reveal/pullRequest?prid=4684568)